Buscorepuestos/buscorepuestos-web
# Feature/br 357 searchbar

## What type of PR is this? (check all applicable)


## Description
Aplicar el diseño propuesto y el responsive al componente Search-Box


## Related Tickets & Documents
https://linear.app/buscorepuestos/issue/BR-357/searchbar

## QA Instructions, Screenshots, Recordings

Ejecutar `next dev` en la rama `feature/br-357-searchbar` y verificar que el elemento buscador del header cumple con el diseño y es responsive.

Also make sure you check this browser compatibility version for desktop and mobile:

## Checklist

DO NOT send the PR without completing the checklist above:


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## Are there any post deployment tasks we need to perform?
